docs/users-guide/account-settings.md

```diff
@@ -0,0 +1,24 @@
+## Account settings
+
+You can view account settings by clicking on the **gears** icon in the upper right of the main navigation bar, then selecting **Account settings**. Here you'll find three tabs:
+
+- [Profile](#account-profile)
+- [Password](#account-password)
+- [Login history](#account-login-history)
+
+### Account profile
+
+You can set your first and last names, change your email address, and set your language. See our list of [supported languages](../faq/general/what-languages-can-be-used-with-metabase.md#currently-available-languages).
+
+### Account password
+
+You can change your password here. If you're having trouble logging in, see our [How do I reset my password](../faq/using-metabase/how-do-i-reset-my-password.md).
```

Comment:
This might be hidden in situations where an EE instances has enabled SSO, but I'm not 100% sure about that.

Reply:
Yep, if the admin has disabled password logins - but then they're logging in through SSO, so login failure is on that system.
https://www.metabase.com/docs/latest/enterprise-guide/authenticating-with-saml.html#disabling-password-log-in